app/services/post_persister_task_service.rb

```
@@ -1,4 +1,9 @@
 class PostPersisterTaskService < TaskService
+  def initialize(options)
+    super
+    @upload_task = Task.where(:child_task_id => @options[:task].id).first
```

@hsong-rh Should we raise an error if the parent cannot be found.